ca/certificates.go

```diff
@@ -119,6 +119,13 @@ type RootCA struct {
 	// must correspond to the key in the local signer, if provided
 	Cert []byte
 
+	// Intermediates contains a bundle of PEM encoded certificates to append to any issued TLS certificates.
+	// If more than one are provided, the first one must have the same public key and subject as the signing
+	// root certificate (or, if no key is provided, at least one of the root certificates), and the rest
```

Comment:
we require the `Intermediates` to also include the root? Usually certificates don't include the full chain all the way up to the root.